cfme/scripting/release.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+import re
+import sys
+from collections import defaultdict
+
+import click
+import github
+import tabulate
+
+from cfme.utils.conf import docker
+
+
+REPO_NAME = "ManageIQ/integration_tests"
+
+HEADERS = ['PR', 'Labels', 'Author', 'Title']
+FULL_HEADERS = HEADERS + ['DESCRIPTION']
+
+VALID_LABELS = [
+    "ansible",
+    "blackify",
+    "blockers-only",
+    "collections-conversion",
+    "customer-case",
+    "doc",
+    "enhancement",
+    "fix-framework",
+    "fix-locator-or-text",
+    "fix-test",
+    "implement-ssui",
+    "infra-related",
+    "issue-bug",
+    "issue-rfe",
+    "LegacyBranch",
+    "manual",
+    "new-test-or-feature",
+    "Nuage",
+    "other",
+    "py3-compat",
+    "rc-regression-fix",
+    "Redfish",
+    "requirements",
+    "RHV",
+    "sprout",
+    "tech-debt",
+    "test-automation",
+    "test-cleanup",
+    "widgetastic-conversion",
+]
+
+IGNORED_LABELS = ['lint-ok', 'WIP-testing']
+
+PR_LINK = "[{pr}](https://github.com/ManageIQ/integration_tests/pull/{pr})"
```

Comment:
I think this should use `REPO_NAME` in its definition.

Reply:
talked about it offline, not worth it for the messiness of two-stage formatting.